packages/react-draggable-dialog/src/components/DraggableDialogHandle/DraggableDialogHandle.types.ts

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+export type DraggableDialogHandleProps = React.HTMLAttributes<HTMLElement> & {
+  /**
+   * The element that will act as the handle for dragging the dialog.
+   * Only one child is allowed.
+   * @required
+   */
+  children: JSX.Element;
```

Comment:
using `JSX` global namespace is not the most robust approach for specifying types. 

can we use `React.ReactElement` ?

Reply:
Fixed.